jekyll/_cci2/workflows.adoc

```diff
@@ -34,35 +34,45 @@ For example, if only one job in a workflow fails, you will know it is failing in
 Workflows may appear with one of the following states:
 
 [.table.table-striped]
-[cols=2*, options="header", stripes=even]
+[cols=3*, options="header", stripes=even]
 |===
-| State | Description
+| State | Description | Terminal State
 
 | RUNNING
 | Workflow is in progress
+| No
 
 | NOT RUN
 | Workflow was never started
+| No
 
 | CANCELED
 | Workflow was canceled before it finished
+| Yes
 
 | FAILING
 | A job in the workflow has failed
+| No
 
 | FAILED
 | One or more jobs in the workflow failed
+| Yes
 
 | SUCCESS
 | All jobs in the workflow completed successfully
+| Yes
 
 | NEEDS APPROVAL
```

Comment:
~ just a note that in the backend and APIs this state is `ON_HOLD`, but the UI started renaming it as of 2 months ago. Should we include both statuses? Or a separate column for the UI status?

```suggestion
| NEEDS APPROVAL (UI) / ON HOLD
```